server/lib/nl/config_builder/timeline.py

```diff
@@ -96,21 +98,27 @@ def ranked_timeline_collection_block(builder: base.Builder,
 
 def single_place_single_var_timeline_block(column, place: Place, sv_dcid: str,
                                            sv2thing: types.SV2Thing,
-                                           date: types.Date, cv: ChartVars):
+                                           single_date: types.Date,
+                                           date_range: types.Date,
+                                           cv: ChartVars):
   """A column with two charts, main stat var and per capita"""
   stat_var_spec_map = {}
 
   title = base.decorate_chart_title(title=sv2thing.name[sv_dcid], place=place)
 
   # Line chart for the stat var
   sv_key = sv_dcid
-  date_string = get_date_string(date)
+  date_string = get_date_string(single_date)
   if date_string:
     sv_key += f'_{date_string}'
-  facet_id = _get_facet_id(sv_dcid, date, cv)
+  facet_id = _get_facet_id(sv_dcid, single_date or date_range, cv)
   if facet_id:
     sv_key += f'_{facet_id}'
   tile = Tile(type=Tile.TileType.LINE, title=title, stat_var_key=[sv_key])
+  if date_range:
+    start_date, end_date = get_date_range(date_range)
```

Comment:
hmmm. related to the comment earlier on this function, here it seems you're returning a tuple but the function comment makes it seem to expect only one date?

Reply:
hm, I feel like the function comment indicates 2 strings are returned? Or do you have any suggestions to improve this? 

"""
# Gets the date range as 2 ISO-8601 formatted strings (i.e., YYYY-MM or YYYY).
# First string is the start date and the second string is the end date.
"""

Reply:
Turns out, in this PR you have two of these functions (``get_date_range()``)? One is in ``lib/nl/detection/date.py`` (this is the one that you were referring to) and the other one seems to be in ``... lib/nl/common/utils.py`` (that's the one I was referring to). Perhaps a good idea to disambiguate them by changing the function name for one of them?

Reply:
ohh you're right, that IS confusing. Updated the one in date.py to `get_date_range_strings`